### ISSUE TITLE
Adding non-steam games, and games in library I don't have ?

### ISSUE THREAD

**Issue by TheMiziko** [Depressurizer/Depressurizer#126]
Hello everyone !
Is there any way to easily manage non-steam games, or add them in the program so that they will be imported to steam ?

And why do I have some games in my library that I never owned/don't have on pc ?

**Comment by Theo47** [comment]
Easiest way would be to do the opposite. Add the non-steam games directly to steam. Then you can import them from steam to depressurizer if you want.

Can you give some examples of what games appear on your library that you don't own? Also a link to your steam profile.

**Comment by TheMiziko** [comment]
@Theo47 For example, this is view from my steam : https://gyazo.com/820d7861f71cf4c643c8016e452d984f

My steam profile : http://steamcommunity.com/id/Sailianna/

And how it looks from the program : https://gyazo.com/dfeec3f7decdbcf362267300231dc455 ( I only took the games that are not showing in the library, there's currently 5/6 of them If I categorized them correctly

I don't know why are they even imported if I don't have them even bought on steam.

**Comment by Theo47** [comment]
Not sure what is causing this. Can you do one more test for me?

1. Close Steam and Depressurizer.
1. Go to `Steam\userdata\<number>\7\remote` and open `sharedconfig.vdf` with a text editor (notepad++ works best if you have that, alternatively wordpad should suffice).
1. Search for "13140" (that is America's Army game id).
You should see smth like:
>"13140"
>     {
>	     "tags"
>           {
>                      "0"		"(HLTB) 10-20"
>                      "1"		"Not in steam liblary"
>           }
>     }
4. Delete the above text and then save the file.
5. After that open Depressurizer and check if America's Army is still there.

**Comment by TheMiziko** [comment]
@Theo47 That's how my notepad++ looks in this line : https://gyazo.com/4593fb4c64939b2c7a11b4b9e0badf6b

and after the whole process it's still here : https://gyazo.com/1b58efc61eda2ffca067595d901e0797

**Comment by Theo47** [comment]
On Depressurizer can you press Profile->Import Steam Config?

Can you reopen sharedconfig.vdf and see if that text is back?

p.s: If you saved with depressurizer then you'll need to redo the steps on the previous post again first. (close depressurizer and steam, delete selected text on sharedconfig.vdf, then open depressurizer, press Profile->Import Steam Config and check if game is still there.)

**Comment by TheMiziko** [comment]
@Theo47 Yes, the game is back in the library. I also tried to completely delete all information from Depressurizer and %appdata% of Depressurizer to make sure everything is clear. Unfortunately it didn't help

**Comment by Theo47** [comment]
Thought this may be related to leftover data from steam's free weekends but in that case i'd expect that deleting it in sharedconfig.vdf would remove the entries. So I've run out of ideas too. It seems Steam reports i have 1111 games while depressurizer reports 1144 games so this is happening to me too. I'll have to take a closer look at the whole process to see when they are added and why.

In the meantime, as a workaround, since you already identified the extra games, right click on them and select "remove game". That should remove them from Depressurizer. If they keep coming back on each launch then go to Profile->Edit Profile, switch to the ignored games tab and check "Ignore games when removing them".
